share/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+if [ $# -eq 0 ]
+  then
+    echo "Path to package need to be specify"
```

Comment:
Validate more preciously

* If command is bad report help signature`

`package.sh [url] [[instanceUrl1] [instanceUrl2]...]`

Reply:
"specified" :)